pkg/fanal/secret/builtin-rules.go

```diff
@@ -158,6 +158,15 @@ var builtinRules = []Rule{
 		Regex:    MustCompile(`glpat-[0-9a-zA-Z\-\_]{20}`),
 		Keywords: []string{"glpat-"},
 	},
+	{
+	        ID:              "hugging-face-access-token",
+	        Category:        types.SecretRuleCategory("HuggingFace"),
+	        Severity:        "CRITICAL",
+	        Title:           "Hugging Face Access Token",
+	        Regex:           MustCompile(`hf_[A-Za-z0-9]{39}`),
+	        SecretGroupName: "secret",
```

Comment:
You don't use groups in regex.
This field is needed to take/hide part of the found string.
If I understand correctly, you need to hide the full line, so you can delete it.

Reply:
What do you mean by not using groups? I admit I'm not really familiar with Go as a language and was trying to reverse engineer what to do based on similar components within the codebase.

Reply:
I meant named regex groups ([example](https://regex101.com/r/RBC6wb/1)) - https://www.regular-expressions.info/named.html

Reply:
To save your time - i did refactor a bit.
Can you take a look and confirm that it works correctly?

Reply:
Group has been removed in https://github.com/aquasecurity/trivy/pull/6236/commits/5bc92a101e9ce54b62ffac2fbe8fbc59f3c9e3c4